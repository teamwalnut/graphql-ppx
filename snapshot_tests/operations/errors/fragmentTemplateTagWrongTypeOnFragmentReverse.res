%graphql(`
  fragment ListFragmentInner on Lists @ppxConfig(templateTagLocation: "gql", templateTagReturnType: "bool") {
    nullableOfNullable
    nullableOfNonNullable
  }
`)

%graphql(`
  fragment ListFragment on Lists  {
    nullableOfNullable
    ...ListFragmentInner
  }
`)
