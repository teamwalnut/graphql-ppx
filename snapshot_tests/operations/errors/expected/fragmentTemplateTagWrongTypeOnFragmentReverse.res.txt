
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/fragmentTemplateTagWrongTypeOnFragmentReverse.res[0m:[2m11:8-24[0m

   9 [2mâ”‚[0m   fragment ListFragment on Lists  {
  10 [2mâ”‚[0m     nullableOfNullable
  [1;31m11[0m [2mâ”‚[0m     ...[1;31mListFragmentInner[0m
  12 [2mâ”‚[0m   }
  13 [2mâ”‚[0m `)
  14 [2mâ”‚[0m 

  This has type: [1;31mbool[0m
  But this function argument is expecting: [1;33mstring[0m

